test/parallel/test-wasm-simple.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+require('../common');
+
+const assert = require('assert');
+const fixtures = require('../common/fixtures');
+const fs = require('fs');
+
+const buffer = fs.readFileSync(fixtures.path('simple.wasm'));
+
+assert.ok(WebAssembly.validate(buffer), 'Buffer should be valid WebAssembly');
+
+WebAssembly.instantiate(buffer, {}).then((results) => {
+  assert.equal(results.instance.exports.addTwo(10, 20), 30, 'Exported function should add two numbers.');
```

Comment:
Does this pass `make lint`? I think we have a rule against `assert.equal` (in favour of `assert.strictEqual`)

Reply:
updated to fix lint